app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -483,10 +483,6 @@ struct DeviceDelegateOculusVR::State {
         } else {
           VRB_WARN("Undefined hand type in DeviceDelegateOculusVR.");
         }
-
-        // This is always false in Oculus Browser.
-        const bool thumbRest = false;
-        controller->SetButtonState(controllerState.index, ControllerDelegate::BUTTON_OTHERS, 5, thumbRest, thumbRest);
       } else {
```

Comment:
It looks odd to me. This is thumbRest instead of the grip button, and thumbRest is always true. If `thumbRest` will cause problems to you, Neo2 will have it too because it also has `thumbRest`.

Reply:
You can remove this commit now. I already gave a fix at #2521 by defining a new btn state `BUTTON_SQUEEZE`. That makes sense for us to use it in app.